include/cantera/numerics/NonlinearSolver.h

```diff
@@ -6,8 +6,8 @@
  */
 
 /*
- *  $Date$
- *  $Revision$
+ *  $Date: 2014-01-03 14:57:40 -0600 (Fri, 03 Jan 2014) $
+ *  $Revision: 2644 $
```

Comment:
This change should be removed since it will just make it more difficult to bring the branch up to the present state.